services/src/main/java/org/keycloak/protocol/saml/SamlProtocolUtils.java

```diff
@@ -132,22 +132,6 @@ public static PublicKey getEncryptionKey(ClientModel client) throws Verification
         return getPublicKey(client, SamlConfigAttributes.SAML_ENCRYPTION_CERTIFICATE_ATTRIBUTE);
     }
 
-    /**
-     * Returns private key used to decrypt SAML assertions encrypted by 3rd party SAML IDP
```

Comment:
Minor: Probably some java "imports" can be removed from this class once this method is removed?